includes/data-stores/interfaces/wc-customer-data-store-interface.php

```diff
@@ -0,0 +1,48 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WC Customer Data Store Interface
+ *
+ * Functions that must be defined by customer store classes.
+ *
+ * @version  2.7.0
+ * @category Interface
+ * @author   WooThemes
+ */
+interface WC_Customer_Data_Store_Interface {
+
+	/**
+	 * Deletes a customer from the database.
+	 *
+	 * @param WC_Customer
+	 * @param int|null $reassign Who to reassign posts to.
+	 */
+	public function delete( &$data, $reassign = null );
```

Comment:
The session and DB store's both implement
`public function delete( &$customer, $force_delete = true, $reassign = null )` with three parameters instead of the two here.

The PHP manual also states (http://php.net/manual/en/language.oop5.interfaces.php):

`Prior to PHP 5.3.9, a class could not implement two interfaces that specified a method with the same name, since it would cause ambiguity. More recent versions of PHP allow this as long as the duplicate methods have the same signature.`.

So I think we will have some problems here defining a different signature.